graphql_client_cli/src/main.rs

```diff
@@ -39,9 +43,15 @@ enum Cli {
     #[structopt(name = "generate")]
     Generate {
         // should be a glob
-        paths: String,
         #[structopt(parse(from_os_str))]
-        schema: PathBuf,
+        query_path: PathBuf,
+        #[structopt(parse(from_os_str))]
+        schema_path: PathBuf,
+        selected_operation: String,
```

Comment:
I think we could make selected_operation optional (in the `derive` implementation we just take the first one by default if no operation is selected).

Reply:
@tomhoule 
I think selected_option is also used as module name.
So we couldn't let selected_option be optional.